gnat2goto/driver/tree_walk.adb

```diff
@@ -4312,10 +4307,11 @@ package body Tree_Walk is
       if Defining_Entity (N) = Stand.Standard_Standard or else
         Unit_Name = "system%s"
       then
-         null;
-         --   At the moment Standard or System are not processed - to be done
+         --  At the moment Standard or System are not processed: TODO
+         Put_Line (Standard_Error,
+         "At the moment Standard or System library units are not processed.");
       else
-
+         --  Withed library unit that has no separate declaration
```

Comment:
I do not understand this comment. These withed library units do have library-level declarations.  Any user defined library-level unit must have an explicit declaration whether it is a package, subprogram declaration or just a subprogram body.  This is what I tried to explain in the comments under each case.

A more general comment preceding the case statement might be
"--  Handle all other withed library unit declarations"

Reply:
Agreed, changed.